[2023-04-28T07:32:56.887+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: stock_analyzer.get_data manual__2023-04-28T07:32:28.396226+00:00 [queued]>
[2023-04-28T07:32:56.927+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: stock_analyzer.get_data manual__2023-04-28T07:32:28.396226+00:00 [queued]>
[2023-04-28T07:32:56.928+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-04-28T07:32:56.928+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2023-04-28T07:32:56.928+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-04-28T07:32:56.968+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): get_data> on 2023-04-28 07:32:28.396226+00:00
[2023-04-28T07:32:57.001+0000] {standard_task_runner.py:55} INFO - Started process 675 to run task
[2023-04-28T07:32:57.016+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'stock_analyzer', 'get_data', 'manual__2023-04-28T07:32:28.396226+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/main.py', '--cfg-path', '/tmp/tmp4fnj71i3']
[2023-04-28T07:32:57.017+0000] {standard_task_runner.py:83} INFO - Job 6: Subtask get_data
[2023-04-28T07:32:57.133+0000] {task_command.py:388} INFO - Running <TaskInstance: stock_analyzer.get_data manual__2023-04-28T07:32:28.396226+00:00 [running]> on host e48c557aa0b5
[2023-04-28T07:32:57.217+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=stock_analyzer
AIRFLOW_CTX_TASK_ID=get_data
AIRFLOW_CTX_EXECUTION_DATE=2023-04-28T07:32:28.396226+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-28T07:32:28.396226+00:00
[2023-04-28T07:32:57.220+0000] {connection.py:287} INFO - Snowflake Connector for Python Version: 3.0.3, Python Version: 3.7.16, Platform: Linux-5.15.90.1-microsoft-standard-WSL2-x86_64-with-debian-11.6
[2023-04-28T07:32:57.221+0000] {connection.py:990} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2023-04-28T07:32:57.221+0000] {connection.py:1007} INFO - Setting use_openssl_only mode to False
[2023-04-28T07:33:00.812+0000] {cursor.py:800} INFO - query: [SELECT * FROM users]
[2023-04-28T07:33:00.933+0000] {cursor.py:813} INFO - query execution done
[2023-04-28T07:33:00.934+0000] {cursor.py:956} INFO - Number of results in first chunk: 5
[2023-04-28T07:33:00.987+0000] {connection.py:586} INFO - closed
[2023-04-28T07:33:01.046+0000] {connection.py:589} INFO - No async queries seem to be running, deleting session
[2023-04-28T07:33:01.195+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-04-28T07:33:01.225+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=stock_analyzer, task_id=get_data, execution_date=20230428T073228, start_date=20230428T073256, end_date=20230428T073301
[2023-04-28T07:33:01.272+0000] {local_task_job.py:208} INFO - Task exited with return code 0
